contrib/question-container-debugging.sh

```diff
@@ -78,7 +78,7 @@ END
 
 echo_and_eval() {
     echo "$@"
-    eval "$@"
+    "$@"
```

Comment:
According to shellcheck, this is more correct. https://www.shellcheck.net/wiki/SC2294